spec/helpers/errors_spec.cr

```diff
@@ -0,0 +1,31 @@
+require "../env_helper"
+require "kilt"
+
+Spectator.describe "error_redirect_helper" do
+  it "shows next steps on embed page errors" do
+    current_url = "/embed/IeQT18gaB-c?si=YxBQzH-GBSTS4vBS"
+    test_env = test_env current_url
+    test_env.set "current_page", current_url
+
+    html = error_redirect_helper(test_env)
+    expect(html).to eq "<p style=\"margin-bottom: 4px;\">After which you should try to: </p>\n<ul>\n  \n    <li>\n      <a href=\"/embed/IeQT18gaB-c?si=YxBQzH-GBSTS4vBS\">Refresh</a>\n    </li>\n  \n    <li>\n      <a href=\"/redirect?referer=/embed/IeQT18gaB-c?si=YxBQzH-GBSTS4vBS\">Switch Invidious Instance</a>\n    </li>\n  \n    <li>\n      <a href=\"https://youtube.com/embed/IeQT18gaB-c?si=YxBQzH-GBSTS4vBS\">Go to YouTube</a>\n    </li>\n  \n</ul>\n"
```

Comment:
For a second I though that we had an URL handling issue, but it's just that you're not passing an encoded URL like the HTTP server handlers would do (see how in the "Switch Invidious Instance" URL nothing after `referer=` is URL encoded?).

It also made me realize that the `si=` parameter isn't removed from the "Watch on Youtube URL" (for the record, this is used to track who shared the video)